### PR TITLE
add logs to beam skimmer

### PR DESCRIPTION
**Pull request by bhavyalatha26**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1688)

### CODE REVIEW

File: src/main/scala/beam/router/BeamSkimmer.scala [hunk 1]
```diff
@@ -57,12 +57,14 @@ class BeamSkimmer @Inject()(val beamConfig: BeamConfig, val beamServices: BeamSe
   private def initialPreviousSkims(): TrieMap[(Int, BeamMode, Id[TAZ], Id[TAZ]), SkimInternal] = {
     if (beamConfig.beam.warmStart.enabled) {
       try {
-        skimsFilePath
+        val previousSkims = skimsFilePath
           .map(BeamSkimmer.readCsvFile)
           .getOrElse(TrieMap.empty)
+        logger.info(s"Previous skims successfully loaded from path '${skimsFilePath.getOrElse(",")}'")
```
Comment (JustinPihony) [inline src/main/scala/beam/router/BeamSkimmer.scala]: Why is the `getOrElse` a comma? Did you mean to do a `mkString(",")`?

Comment (bhavyalatha26) [reply src/main/scala/beam/router/BeamSkimmer.scala]: could be empty string .. 
`getOrElse("")`

Comment (JustinPihony) [reply src/main/scala/beam/router/BeamSkimmer.scala]: Maybe be more explicit and make it something like "Not Found"?

**Comment by JustinPihony** [comment]
Should this be merging into master or develop?

**Pull request closed without merging**